go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -365,15 +366,16 @@ func newOvnAddressSet(nbClient libovsdbclient.Client, name string, ips []net.IP)
 	} else {
 		ops := make([]ovsdb.Operation, 0, 2)
 		timeout := types.OVSDBWaitTimeout
-		ops = append(ops, ovsdb.Operation{
-			Op:      ovsdb.OperationWait,
-			Timeout: &timeout,
-			Table:   "Address_Set",
-			Where:   []ovsdb.Condition{{Column: "name", Function: ovsdb.ConditionEqual, Value: name}},
-			Columns: []string{"name"},
-			Until:   "!=",
-			Rows:    []ovsdb.Row{{"name": name}},
-		})
+		condition := model.Condition{
+			Field:    &addrSet.Name,
+			Function: ovsdb.ConditionEqual,
+			Value:    addrSet.Name,
```

Comment:
There is a diff between `hashedAddressSet(name)` and `name`. I do not understand how safe it is to use one vs the other in this context.

Reply:
actually `hashedAddressSet(name)` is what set as name for `addrSet`.
https://github.com/ovn-org/ovn-kubernetes/blob/6c11d719bb867dd12c3c17a53c548fa1c627df75/go-controller/pkg/ovn/address_set/address_set.go#L340
 so shouldn't we use this to check for object existence ?

Reply:
The real name is stored in the external_ids. I think the "name" field has a limit on chars so we use the hashed name there. Looks OK to me.